tests/regression/run_regression.py

```diff
@@ -0,0 +1,65 @@
+"""Checks export against known dataset.
+
+This is intended to be primarily run as part of CI only, and does not leverage
+pytest due to the requirements for AWS secrets/federated tokens/network 
+connectivity."""
```

Comment:
nit: I have not read rest of PR - so forgive me if this is answered elsewhere - but the "due to" is not immediately obvious to me why pytest wouldn't work. Maybe just like, one more sentence on that.

Reply:
I guess - my thought was 'if you are running pytest locally, and you want this to work, you either have to be signed into the BCH VPN, and have an auth token, or we have to open a much less secure way to use these credentials from another source'. The second is a non starter. The first is irritating enough that I kind of want to not have this picked up during a normal pytest run.

I could make this pytest-based and have this directory be normally excluded? but then you could run it manually with pytest if so inclined?

Reply:
I don't need it to be pytest, was just confused by the docstring here.

How about something like "this isn't in pytest because it isn't run as part of a test suite, and it's awkward to connect correctly to AWS (see ci.yaml)" -- I think I was struck by the "due to the requirements" making it sound impossible, but the real answer is "oy it's too much of a pain"

Reply:
something in that flavor should work, yeah

Reply:
OK, this is more explicit now.